### ISSUE TITLE
Train NER-models

### ISSUE THREAD

**Issue by veleda** [Sjofartsdirektoratet/NLP_PoC#11]
Using classification from ET ontology to train NER-models; information about subclasses.

input: regulation text --> entities --> classification (super class) --> CSV --> output: ontology --> Q&A --> prod ontology

**Comment by veleda** [comment]
How to classify entities that are unknown for the test set/trained models?

* Look for open and pre-trained models online.
* spaCy NER models?

**Comment by veleda** [comment]
Set up a meeting with Miriam.

**Comment by veleda** [comment]
Get input from Miriam.
